### PR TITLE
Removed use of PyMEL

### PR DESCRIPTION
**Pull request by UnaiM**
All functionality previously achieved via PyMEL now uses maya.cmds and maya.mel instead. This provides substantially faster loading times. Several module imports were cleaned up too.

### CODE REVIEW

File: python/tk_maya/menu_generation.py [hunk 1]
```diff
@@ -110,34 +106,34 @@ def _add_context_menu(self):
...
+    def _jump_to_sg(self, *args):
         """
         Jump to shotgun, launch web browser
         """        
         url = self._engine.context.shotgun_url
         QtGui.QDesktopServices.openUrl(QtCore.QUrl(url))
         
         
-    def _jump_to_fs(self):
+    def _jump_to_fs(self, *args):
```
[the remark this answers is not in the document]
Comment (UnaiM) [reply python/tk_maya/menu_generation.py]: Blank lines are the same as in #68.

File: python/tk_maya/menu_generation.py [hunk 2]
```diff
@@ -110,34 +106,34 @@ def _add_context_menu(self):
...
 
         # divider (apps may register entries below this divider)
-        pm.menuItem(divider=True, parent=ctx_menu)
         
+        cmds.menuItem(divider=True, parent=ctx_menu)
         return ctx_menu
                         
                         
-    def _jump_to_sg(self):
+    def _jump_to_sg(self, *args):
```
[the remark this answers is not in the document]
Comment (UnaiM) [reply python/tk_maya/menu_generation.py]: Blank lines are the same as in #68.

File: python/tk_maya/menu_generation.py [hunk 3]
```diff
@@ -71,8 +67,8 @@ def create_menu(self, *args):
                      # mark as a favourite item
                      cmd.favourite = True
 
-        pm.menuItem(divider=True, parent=self._menu_handle)
         
+        cmds.menuItem(divider=True, parent=self._menu_handle)
```
[the remark this answers is not in the document]
Comment (UnaiM) [reply python/tk_maya/menu_generation.py]: Blank lines are the same as in #68.

**Comment by pscadding** [comment]
Hey @UnaiM , I've just looked over this. Unfortunately, I didn't spot it sooner, and I've actually reimplemented the changes you made over here:
https://github.com/shotgunsoftware/tk-maya/pull/79/files

Still, this was useful as I cross-checked my changes against yours. There are some subtle differences but functionally it is mostly the same. I'm going to close this out, but we'll credit you as well in the release.

**Comment by UnaiM** [comment]
No worries @pscadding! Thanks!

**Pull request closed without merging**